docs/plugins/harbor.zh.md

```diff
@@ -1,55 +1,285 @@
+换言之，当前部署方式会自动完成 PostgreSQL 和 Redis 的部署，但是同时需要注意 PostgreSQL 和 Redis 并不是高可用的。
+
+- **Service** (`kubectl get service -n harbor`)
+
+```shell
+NAME                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+harbor              NodePort    10.99.177.6      <none>        80:30002/TCP        4m17s
+harbor-core         ClusterIP   10.106.220.239   <none>        80/TCP              4m17s
+harbor-database     ClusterIP   10.102.102.95    <none>        5432/TCP            4m17s
+harbor-jobservice   ClusterIP   10.98.5.49       <none>        80/TCP              4m17s
+harbor-portal       ClusterIP   10.105.115.5     <none>        80/TCP              4m17s
+harbor-redis        ClusterIP   10.104.100.167   <none>        6379/TCP            4m17s
+harbor-registry     ClusterIP   10.106.124.148   <none>        5000/TCP,8080/TCP   4m17s
+```
+
+可以看到默认情况下 Harbor 通过 NodePort 方式暴露服务时，端口是 30002。
+
+- **PersistentVolumeClaim** (`kubectl get pvc -n harbor`)
+
+```shell
+NAME                              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+data-harbor-redis-0               Bound    pvc-5b6b5eb4-c40d-4f46-8f19-ff3a8869e56f   1Gi        RWO            standard       5m12s
+database-data-harbor-database-0   Bound    pvc-d7ccaf1f-c450-4a16-937a-f55ad0c7c18d   1Gi        RWO            standard       5m12s
+harbor-jobservice                 Bound    pvc-9407ef73-eb65-4a56-8720-a9ddbcb76fef   1Gi        RWO            standard       5m13s
+harbor-registry                   Bound    pvc-34a2b88d-9ff2-4af4-9faf-2b33e97b971f   5Gi        RWO            standard       5m13s
+```
+
+从这里可以看到 Harbor 所需要的存储卷有4个，其中也包括了 PostgreSQL 和 Redis 的存储。
+
+- **PersistentVolume** (`kubectl get pv`)
+
+```shell
+pvc-34a2b88d-9ff2-4af4-9faf-2b33e97b971f   5Gi        RWO            Delete           Bound         harbor/harbor-registry                    standard                5m22s
+pvc-5b6b5eb4-c40d-4f46-8f19-ff3a8869e56f   1Gi        RWO            Delete           Bound         harbor/data-harbor-redis-0                standard                5m22s
+pvc-9407ef73-eb65-4a56-8720-a9ddbcb76fef   1Gi        RWO            Delete           Bound         harbor/harbor-jobservice                  standard                5m22s
+pvc-d7ccaf1f-c450-4a16-937a-f55ad0c7c18d   1Gi        RWO            Delete           Bound         harbor/database-data-harbor-database-0    standard                5m22s
+```
+
+我们并没有配置 StorageClass，所以这里用的是集群内的 default StorageClass 完成的 pv 创建。
+
+在我的环境里，default StorageClass 如下(`kubectl get storageclass`)：
+
+```shell
+NAME                 PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+standard (default)   k8s.io/minikube-hostpath   Delete          Immediate           false                  20h
+```
+
+这个 StorageClass 对应的 Provisioner 会以 hostPath 的方式提供 pv。
+
+到这里，我们就可以通过 http://127.0.0.1:3002 访问到 Harbor 登录页面了，如下：
```

Comment:
Maybe [http://127.0.0.1:3002](http://127.0.0.1:3002) is better.

<img width="686" alt="image" src="https://user-images.githubusercontent.com/36830265/185305217-0e7a6f62-9a4b-47d2-b5b3-a968426408d2.png">